promenade/templates/common/etc/dnsmasq.d/upstream-dns

```diff
@@ -0,0 +1,9 @@
+no-hosts
+no-resolv
+
+strict-order
+
+server={{ config['Network']['cluster_dns'] }}
```

Comment:
For hosts that have not yet joined the cluster, will they be able to reach the cluster DNS to resolve skyDNS names?